config/clusters/pangeo-hubs/common.values.yaml

```diff
@@ -10,6 +10,9 @@ basehub:
       # Name of Google Filestore share
       baseShareName: /homes/
   jupyterhub:
+    scheduling:
+      userScheduler:
+        enabled: true
```

Comment:
doesn't this hub run on GCP? Why do we need the user scheduler given that this hub is (I assume) using the `Optimize Utilization` setup you describe?

Reply:
@choldgraf great catch! Fixed.